tests/shadow/test_shadow_measurement.py

```diff
@@ -49,19 +44,13 @@ def get_x_basis_circuit(wires, shots, interface="autograd", device="default.qubi
     """
     Return a QNode that prepares the |++..+> state and performs a classical shadow measurement
     """
-    if device is not None:
-        dev = qml.device(device, wires=wires, shots=shots)
-    else:
-        dev = qml.device("default.qubit", wires=wires, shots=shots)
-
-        # make the device call the superclass method to switch between the general qubit device and device specific implementations (i.e. for default qubit)
-        dev.classical_shadow = super(type(dev), dev).classical_shadow
+    dev = qml.device(device, wires=wires, shots=shots)
 
     @qml.qnode(dev, interface=interface)
     def circuit():
         for wire in range(wires):
             qml.Hadamard(wire)
-        return qml.classical_shadow(wires=range(wires))
+        return qml.classical_shadow(wires=range(wires), seed_recipes=False)
```

Comment:
Why was this change necessary?

Reply:
I don't think it was necessary. I was testing some stuff and left it by mistake. Removed 👍